lib/game.rb

```diff
@@ -20,6 +21,11 @@ def user_input
     input
   end
 
+    # def choose_opponent_prompt
+  #   "\nType 'Computer' to play against computer or 'Player' to play against another player\n"
+  # end
+  #^there due to limitation on testing methods with user input.
```

Comment:
How do we choose whether we play a computer or a person?

Reply:
That isn't yet added in this PR. In my comment I mentioned this is the final PR before working on player 2.